packages/terra-slider/src/Slider.module.scss

```diff
@@ -0,0 +1,203 @@
+@import '~terra-mixins/lib/Mixins';
+
+// Themes
+@import './orion-fusion-theme/Slider.module';
+@import './clinical-lowlight-theme/Slider.module';
+
+:local {
+  .slider {
+    align-items: center;
+    display: inline-flex;
+    min-height: 3.143rem;
+  }
+
+  .label {
+    color: var(--terra-slider-label-color, #1c1f21);
+    font-weight: var(--terra-slider-label-font-weight, 700);
+    margin-right: var(--terra-slider-label-margin-right, 0.5rem);
+    text-align: center;
+    white-space: initial;
+    width: 20%;
+  }
+
+  .slider-label {
+    color: var(--terra-slider-sub-label-color, #6f7477);
+    font-size: var(--terra-slider-sub-label-font-size, 0.857rem);
+    font-weight: var(--terra-slider-sub-label-font-weight, 400);
+    text-align: left;
+    white-space: initial;
+  }
+
+  .slider-min-label {
+    margin-right: var(--terra-slider-min-label-margin-right, 0.5rem);
+  }
+
+  .slider-max-label {
+    margin-left: var(--terra-slider-max-label-margin-left, 0.5rem);
+  }
+
+  .input-range {
+    -webkit-appearance: none; /* stylelint-disable-line property-no-vendor-prefix, plugin/no-unsupported-browser-features */
+    background-color: transparent;
+    margin-left: var(--terra-slider-input-range-margin-left, 0.5rem);
+    margin-right: var(--terra-slider-input-range-margin-right, 0.5rem);
+    min-height: 3.143rem;
+    min-width: 17.143rem;
```

Comment:
Do we really want the minimum width to be so large?  Are we considering mobile devices?  It seems like the consumer should be able to control the width by the size of the parent container.

Reply:
Requested @eawww  to double check on this. But I am guessing this is what they need on callouts.

Reply:
Looks like the concern was around the width being too small and the thumb movement being minimal those cases. That's why an optimal min-width is suggested here.